docs/modules/ROOT/pages/whats-new.adoc

```diff
@@ -0,0 +1,37 @@
+= What's New in Hazelcast Platform Operator
+:description: Here are the highlights of what's new and improved in Hazelcast Platform Operator 5.13.0.
```

Comment:
```suggestion
:description: Here are the highlights of what's new and improved in Hazelcast Platform Operator 5.13.
```
I'd just using x.y here as we will add to this page if e.g. 13.1 adds new features